bundles/org.openhab.binding.gpstracker/src/main/java/org/openhab/binding/gpstracker/internal/message/dto/LocationMessage.java

```diff
@@ -53,7 +53,7 @@ public class LocationMessage {
      * Altitude (iOS, Android/integer/meters/optional)
      */
     @SerializedName("alt")
-    private Integer altitude = Integer.MIN_VALUE;
+    private BigDecimal altitude = BigDecimal.ZERO;
```

Comment:
Minor, but three lines up it's still documented as "integer". Just spotted this now.

Reply:
Good catch